apps/boilerplate-react-vite/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@canonical/typescript-config-react",
   "compilerOptions": {
     "baseUrl": "src",
+    "types": ["node", "react", "react-dom"],
```

Comment:
Please add explicit type `devDependencies` for these to the boilerplate `package.json`, it is currently missing them